tests/unit/test_param_escaper.py

```diff
@@ -0,0 +1,108 @@
+from datetime import date, datetime
+import unittest, pytest
+
+from databricks.sql.utils import ParamEscaper, inject_parameters
+
+pe = ParamEscaper()
+
+class TestIndividualFormatters(object):
+
+    # Test individual type escapers
+    def test_escape_number_integer(self):
+        """This behaviour falls back to Python's default string formatting of numbers
+        """
+        assert pe.escape_number(100) == 100
+
+    def test_escape_number_float(self):
+        """This behaviour falls back to Python's default string formatting of numbers
+        """
+        assert pe.escape_number(100.1234) == 100.1234
+
+    def test_escape_string_normal(self):
+        """
+        """
+
+        assert pe.escape_string("golly bob howdy") == "'golly bob howdy'"
+
+    def test_escape_string_that_includes_quotes(self):
+        # Databricks queries support just one special character: a single quote mark
```

Comment:
i don't understand this comment, per the [spark docs](https://spark.apache.org/docs/latest/sql-ref-literals.html) backslash is used for escaping (so it too is a special character)

you might also try this query in Databricks SQL:  
```
select 'cat\'s meow' 
union all
select 'cat\\\'s meow'
union all
select '\U0001F44D \u3042'
```

see also https://github.com/databricks/databricks-sql-python/issues/51

Reply:
Good callout. Removing this comment in dd25478c4e94fddb33884f6d206fd62427e5076c